LogMonitor/src/LogMonitor/Main.cpp

```diff
@@ -288,7 +289,7 @@ int __cdecl wmain(int argc, WCHAR *argv[])
             cmdline += argv[i];
         }
 
-        CreateAndMonitorProcess(cmdline);
+        exitcode = CreateAndMonitorProcess(cmdline);

```

Comment:
It looks like the type of CreateAndMonitorProcess function is DWORD and exitcode is int. 

I reckon we need to do some type conversion and error checks if possible?

Reply:
DWORD to int conversion is handled implicitly, open to running a few tests to see chances of failure